docs/detections/rules-prebuilt.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[rules-prebuilt]]
+== Install Elastic prebuilt rules
+
+:frontmatter-description: Start detections quickly with prebuilt rules designed and updated by Elastic.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+Follow these guidelines to start using the {es-sec-app}'s <<prebuilt-rules, prebuilt rules>>, keep them updated, and make sure they have the data needed to run successfully. 
+
+* <<load-prebuilt-rules>>
+* <<prebuilt-rule-tags>>
+* <<select-all-prebuilt-rules>>
+* <<update-prebuilt-rules>>
+* <<download-prebuilt-rules>>
+* <<rule-prerequisites>>
+
+[NOTE]
+====
+* Apart from the Elastic Endpoint rule, prebuilt rules do not start running by default. You must first install the rules, then enable them.
+
+* You can't modify most settings on Elastic prebuilt rules. You can only edit <<rule-schedule, rule actions>> and <<add-exceptions, add exceptions>>. If you want to modify other settings on a prebuilt rule, you must first duplicate it, then make your changes to the duplicated rule. 
```

Comment:
Should the link to the rule action be `rule-notifications` instead? Right now, it's pointing to the instructions for setting a rule's execution schedule.

```suggestion
* You can't modify most settings on Elastic prebuilt rules. You can only edit <<rule-notifications, rule actions>> and <<add-exceptions, add exceptions>>. If you want to modify other settings on a prebuilt rule, you must first duplicate it, then make your changes to the duplicated rule. 
```